src/tasks/heartbeat/taskheartbeat.cpp

```diff
@@ -36,12 +36,16 @@ void TaskHeartbeat::run() {
 
             int statusCode = request->statusCode();
 
+            logger.debug() << "Guardian network request encountered an issue. "
```

Comment:
Couldn't this be merged with the existing logging on line 32 by addind the status code there?